tasks/relayer/activateIssuer.ts

```diff
@@ -0,0 +1,24 @@
+import { task } from "hardhat/config";
+import { TaskArguments } from "hardhat/types";
+import { DefenderRelaySigner, DefenderRelayProvider } from "@openzeppelin/defender-relay-client/lib/ethers";
+import { AccessTokenVerifier } from "../../src/types";
+
+task("relay:activateIssuers", "Uses OZ Relayer to activate issuers")
+  .addParam("verifiercontract", "Address of the verifier contract")
+  .addParam("newissuer", "Address of the issuer to add")
+  .setAction(async (taskArguments: TaskArguments, hre) => {
+    if (!process.env.OZ_RELAY_API_KEY || !process.env.OZ_RELAY_API_SECRET)
+      throw new Error("Please set OZ_RELAY_API_KEY and OZ_RELAY_API_KEY env vars");
+
+    const credentials = { apiKey: process.env.OZ_RELAY_API_KEY, apiSecret: process.env.OZ_RELAY_API_SECRET };
+    const provider = new DefenderRelayProvider(credentials);
+    const signer = new DefenderRelaySigner(credentials, provider, { speed: "fast" });
+
+    const accessTokenVerifier = <AccessTokenVerifier>(
+      await hre.ethers.getContractAt("AccessTokenVerifier", taskArguments.verifiercontract, signer)
+    );
+    const tx = await accessTokenVerifier.activateIssuers([taskArguments.newissuer]);
```

Comment:
I think we should add a simple checksum validation on the newIssuer before issuing the tx (please ignore if the function call would fail anyway if we pass an invalid issuer)

Reply:
Interestingly, passing in address vars to contract functions don't care about checksum formatting. Checksum checks aren't actually done at low-level, the EVM doesn't care, it's more like a higher level use-case validation to make sure there aren't silly errors when typing or pasting an address when sending money.

Reply:
So from what I understood, would be wise to do a quick check before sending the tx as if there was a copy/paste issue or typo we would set the issuer wrongly, do you agree? It's always complicated to deal with addresses manually bc they are so long and hard to spot any issues with it by naked eye

Reply:
Something like ethers.verifyAddressChecksum: boolean should exist right?

Reply:
IMO it doesn't make a difference because then we're circumventing the check anyway.

I did some more digging to figure out the significance of this but the final sentiment seems to be "to avoid sending transactions to invalid addresses". No one has ever defined what an 'invalid address' actually is, considering that the entire 20-byte address range contains all valid addresses, what makes one invalid?

Anyway I think it's trivial to add the one-liner to make this check, but I still can't justify why this even exists LOL